app_dart/test/request_handlers/postsubmit_luci_subscription_test.dart

```diff
@@ -145,4 +150,24 @@ void main() {
     expect(await tester.post(handler), Body.empty);
     expect(task.status, Task.statusInProgress);
   });
+
+  test('Requests with repo_owner and repo_name update checks', () async {
+    when(mockGithubChecksService.updateCheckStatus(any, any, any)).thenAnswer((_) async => true);
+    final Commit commit = generateCommit(1, sha: '87f88734747805589f2131753620d61b22922822');
+    final Task task = generateTask(
+      4507531199512576,
+      parent: commit,
+    );
+    config.db.values[task.key] = task;
+
+    tester.message = createBuildbucketPushMessage(
+      'COMPLETED',
+      result: 'SUCCESS',
+      builderName: 'Linux Packages',
+      userData:
+          '{\\"task_key\\":\\"${task.key.id}\\", \\"commit_key\\":\\"${task.key.parent?.id}\\", \\"repo_owner\\": \\"flutter\\", \\"repo_name\\": \\"packages\\"}',
```

Comment:
Could you use ''' string ''' instead of all the escaping here?

Reply:
This is to mock the json decoded data:
```
{\"task_key\":\"4507531199512576\", \"commit_key\":\"flutter/flutter/master/87f88734747805589f2131753620d61b22922822\", \"repo_owner\": \"flutter\", \"repo_name\": \"packages\"}
```

Reply:
Ah gotcha. Might be good to add a note if it is expected in that format with escapes.

Reply:
Done.